Makefile.in

```diff
@@ -71,7 +71,9 @@ xmlcgi.pgm: xmlcgi.module $(COMMON)
 	system -q "CRTPGM PGM($(LIBRARY)/$(@:%.pgm=%)) MODULE($(^:%.module=$(LIBRARY)/%)) BNDSRVPGM(QHTTPSVR/QZSRCORE)" && touch $@
 
 xmlstoredp.srvpgm: xmlstoredp.module $(COMMON)
-	system -q "CRTSRVPGM SRVPGM($(LIBRARY)/$(@:%.srvpgm=%)) MODULE($(^:%.module=$(LIBRARY)/%)) EXPORT(*ALL) ACTGRP(*CALLER)" && touch $@
+	(system -q 'CHKOBJ $(LIBRARY)/QSRVSRC *FILE' || system -q 'CRTSRCPF $(LIBRARY)/QSRVSRC') && touch $@
```

Comment:
There's already a rule to make a source physical file: https://github.com/IBM/xmlservice/blob/master/Makefile.in#L107-L108

You should instead, just add `qsrvsrc.srcpf` as a dependency of `xmlstoredp.srvpgm`, like here: https://github.com/IBM/xmlservice/blob/master/Makefile.in#L76